resources/charts/kubernetes/values.yaml

```diff
@@ -0,0 +1,43 @@
+apiserver:
+  apiserverKey: 
+  apiserverCert:
+  serviceAccountPub: 
+  caCert: 
+  etcdClientCaCert: 
+  etcdClientCert: 
+  etcdClientKey: 
+  aggregationCaCert: 
+  aggregationClientCert: 
+  aggregationClientKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  cloudProvider: 
+  etcdServers: 
+  aggregationFlags: 
+  serviceCIDR: 10.0.0.0/24
+  trustedCertsDir: /usr/share/ca-certificates
+controllerManager:
+  serviceAccountKey: 
+  caCert: 
+  caKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  cloudProvider: 
+  serviceCIDR: 10.0.0.0/24
+  podCIDR: 10.0.0.0/16
```

Comment:
@invidian please verify the default values for `podCIDR`

Reply:
@invidian just a need a lgtm on this

Reply:
`10.0.0.0/16` clashes with default value of `serviceCIDR`. I guess we can just re-use Terraform default here: `10.2.0.0/16`, as I cannot find any other reasonable default (in kubeadm, the value must be specified by user and it depends on the network plugin the user choose).